### PR TITLE
Book Library

### PR DESCRIPTION
**Pull request by SamArsanios**
In this Project, we used javascript to build a small book library. The application can:

1. Add a book
2. Delete a book
3. Toggle the read status of the book

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            </th>
+            <th>
+                read
+            </th>
+        </thead>
+        <tbody id="tbody"></tbody>
+
+    </table>
+    <div class="text-center">
+        <button class="btn btn-primary" id="form-btn">Add new book</button>
```
Comment (addod19) [inline index.html:34]: The new book button should trigger an action to show the form, but currently, we have the button and the form showing even without pressing the button. This is not the expected behavior, let us kindly fix that

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light:

Hi @SamArsanios and @dibdas, congratulations on this project :+1: The README looks great but the screenshot image is broken. PR is great :fireworks: 
Let us kindly address the comments below so we can get our project approved :smile: 
- We are missing the `addBookToLibrary` function. The function should take the user's input as an argument/parameter. The application currently is not able to add any book in the browser. Let us kindly fix it
- `Display books` function - We are missing the functionality that loops through the array and displays each book. It should display on the page every book contained in the array. 
Each `Book` displayed has a `button` to `remove` the `Book` from the `Library` array. 
Each `Book` displayed has a `button` to `toggle` the `Book` read status

[Optional]
- It would be great if we do not allow a user to submit empty data, let us kindly add validations
- It would be nice if we implement local storage for data persistence so data remains in the browser even after page refresh

Once these are resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

File: index.html (hunk 1 above)
Comment (bellom) [reply index.html:34]: @SamArsanios Please kindly address this.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @SamArsanios   :clap: 

**Great work making `almost` all the necessary corrections from the previous review.**

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

According to the project requirement, you're supposed to have a `button` that will bring up the form and users should be able to specify the status of the book either read or unread in the form.

image

https://www.theodinproject.com/courses/javascript/lessons/library

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Review by arslanbisharat — APPROVED** [review]
# Approved :clap:

Hello Developer, :wave:

Thank You for submitting this Pull Request. I am thrilled to announce that your project has met all the minimum requirements and now it is approved. :clap:

I am wishing you very good luck with the next project. :smile:

Once again, Congratulations!:joy:

**Best,
Arslan Bisharat(PT-TSE)
Slack: @arslanbisharat**

**Pull request merged**